cmd/soroban-cli/src/commands/network/mod.rs

```diff
@@ -181,7 +180,7 @@ impl Network {
         } else {
             let client = Client::new(&self.rpc_url)?;
             let uri = client.friendbot_url().await?;
-            Uri::from_str(&format!("{uri:?}?addr={addr}")).map_err(|e| {
+            Uri::from_str(&format!("{}?addr={addr}", uri.trim_matches('"'))).map_err(|e| {
```

Comment:
I think we should fix this in the client. The `friendbot_url` function shouldn't be returning a URL that is surrounded by quotes.

Reply:
Just realized that the issue was with the debug string.